kcidb/misc.py

```diff
@@ -194,6 +195,32 @@ def non_negative_int_or_inf(string):
     return value
 
 
+def iso_timestamp(string):
+    """
+    Parse an ISO-8601 timestamp out of a string, assuming local timezone, if
+    not specified. Matches the argparse type function interface.
+
+    Args:
+        string: The string to parse.
+
+    Returns:
+        The non-negative integer, or  parsed out of the string.
```

Comment:
Incoherent `Returns:`

Should be something like: `The timestamp parsed out of the string.`